src/sbt-test/missinglink/do-not-fail-on-conflicts/build.sbt

```
@@ -0,0 +1,18 @@
+inThisBuild(Def.settings(
+  version := "0.1.0",
+  scalaVersion := "2.12.8",
+))
+
+lazy val `ignore-source-package` = project
```

```suggestion
lazy val `do-not-fail-on-conflicts` = project
```
to match the test name. (I suspect a copy-paste error.)